growthstreet_zoo/name_generator.py

```diff
@@ -0,0 +1,24 @@
+import random
+import math
+
+
+adjs = ['Autumn', 'Hidden', 'Bitter', 'Misty', 'Silent', 'Empty', 'Dry', 'Dark', 'Summer', 'Icy', 'Delicate', 'Quiet',
```

Comment:
Obviously this is a bit of fun, so I like it!  But be careful of building things not in the scope.  It's an easy trap to fall into, always prioritise getting the requirements right first.